go.mod

```diff
@@ -0,0 +1,36 @@
+module github.com/pcelvng/task-tools
+
+require (
+	github.com/bitly/go-hostpool v0.0.0-20171023180738-a3a6125de932 // indirect
+	github.com/buger/jsonparser v0.0.0-20181115193947-bf1c66bbce23
+	github.com/davecgh/go-spew v1.1.1
+	github.com/dustinevan/chron v1.0.0 // indirect
+	github.com/go-ini/ini v1.42.0 // indirect
+	github.com/go-sql-driver/mysql v1.4.1
+	github.com/golang/snappy v0.0.1 // indirect
+	github.com/google/go-cmp v0.2.0
+	github.com/jbsmith7741/go-tools v0.1.0
+	github.com/jbsmith7741/trial v0.2.1
+	github.com/jbsmith7741/uri v0.2.0
+	github.com/json-iterator/go v1.1.6
+	github.com/lib/pq v1.0.0
+	github.com/minio/minio-go v6.0.14+incompatible
+	github.com/mitchellh/go-homedir v1.1.0 // indirect
+	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
+	github.com/modern-go/reflect2 v1.0.1 // indirect
+	github.com/nsqio/go-nsq v1.0.7
+	github.com/nwingert/task-tools v0.3.0 // indirect
+	github.com/pcelvng/task v0.1.1-0.20181210204914-9f2210538813
+	github.com/pelletier/go-toml v1.3.0
+	github.com/pkg/errors v0.8.1
+	github.com/robfig/cron v0.0.0-20180505203441-b41be1df6967
+	github.com/smartystreets/goconvey v0.0.0-20190330032615-68dc04aab96a // indirect
+	github.com/stretchr/testify v1.3.0
+	golang.org/x/crypto v0.0.0-20190404164418-38d8ce5564a5 // indirect
+	golang.org/x/net v0.0.0-20190403144856-b630fd6fe46b // indirect
+	google.golang.org/appengine v1.5.0 // indirect
+	gopkg.in/BurntSushi/toml.v0 v0.3.1
+	gopkg.in/dustinevan/chron.v0 v0.0.0
+	gopkg.in/ini.v1 v1.42.0 // indirect
+	gopkg.in/jbsmith7741/uri.v0 v0.2.0
```

Comment:
We might want to go through the project and replace imports for gopkg.in/jbsmith7741/uri.v0 with github.com/jbsmith7741/uri. I don't think we need the gopkg.in packages with go modules.